frontend/app/routes/_public+/apply+/$id+/demographics-part2.tsx

```diff
@@ -54,11 +54,11 @@ export async function action({ request, params }: ActionFunctionArgs) {
   const { id } = await applyFlow.loadState({ request, params });
 
   const formSchema = z.object({
-    gender: z.string({ required_error: 'empty-radio' }),
+    gender: z.string().trim().optional(),
     otherGender: z.string().min(1, { message: 'empty-field' }).optional(),
-    mouthPainType: z.string({ required_error: 'empty-radio' }),
-    lastDentalVisitType: z.string({ required_error: 'empty-radio' }),
-    avoidedDentalCareDueToCost: z.string({ required_error: 'empty-radio' }),
+    mouthPainType: z.string().trim().optional(),
+    lastDentalVisitType: z.string().trim().optional(),
+    avoidedDentalCareDueToCost: z.string().trim().optional(),
   });
```

Comment:
These validators will allow for all-whitespace strings. Was that intentional?

Reply:
i followed what was done on other pages, we will fix in a follow up PR, for now we just want to get the question to the page and available for use.